desktop/src/main/java/bisq/desktop/main/offer/AtomicOfferView.java

```diff
@@ -0,0 +1,753 @@
+            tradeFeeVisibleListener;
+    private ChangeListener<String> tradeCurrencyCodeListener, errorMessageListener, volumeListener;
+    private EventHandler<ActionEvent> paymentAccountsComboBoxSelectionHandler;
+    private OfferView.CloseHandler closeHandler;
+
+    protected int gridRow = 0;
+    private final List<Node> editOfferElements = new ArrayList<>();
+    private boolean isActivated;
+    private InfoInputTextField volumeInfoInputTextField;
+    private AutoTooltipSlideToggleButton tradeFeeInBtcToggle, tradeFeeInBsqToggle;
+
+    @Setter
+    private OfferView.OfferActionHandler offerActionHandler;
+
+
+    ///////////////////////////////////////////////////////////////////////////////////////////
+    // Constructor, lifecycle
+    ///////////////////////////////////////////////////////////////////////////////////////////
+
+    public AtomicOfferView(M model,
+                           Navigation navigation,
+                           OfferDetailsWindow offerDetailsWindow) {
+        super(model);
+
+        this.navigation = navigation;
+        this.offerDetailsWindow = offerDetailsWindow;
+    }
+
+    @Override
+    protected void initialize() {
+        addScrollPane();
+        addGridPane();
+        addPaymentGroup();
+        addAmountPriceGroup();
+        addOptionsGroup();
+
+        createListeners();
+
+        paymentAccountsComboBox.setConverter(GUIUtil.getPaymentAccountsComboBoxStringConverter());
+        paymentAccountsComboBox.setButtonCell(GUIUtil.getComboBoxButtonCell(Res.get("shared.selectTradingAccount"),
+                paymentAccountsComboBox, false));
+        paymentAccountsComboBox.setCellFactory(model.getPaymentAccountListCellFactory(paymentAccountsComboBox));
+
+        doSetFocus();
+    }
+
+    protected void doSetFocus() {
+        GUIUtil.focusWhenAddedToScene(amountTextField);
+    }
+
+    @Override
+    protected void activate() {
+        if (model.getDataModel().isTabSelected)
+            doActivate();
+    }
+
+    protected void doActivate() {
+        if (isActivated) {
+            return;
+        }
+        isActivated = true;
+        paymentAccountsComboBox.setPrefWidth(250);
+
+        addBindings();
+        addListeners();
+
+        amountDescriptionLabel.setText(model.getAmountDescription());
+
+        paymentAccountsComboBox.setItems(model.getDataModel().getPaymentAccounts());
+        paymentAccountsComboBox.getSelectionModel().select(model.getPaymentAccount());
+
+        boolean currencyForMakerFeeBtc = model.getDataModel().isCurrencyForMakerFeeBtc();
+        tradeFeeInBtcToggle.setSelected(currencyForMakerFeeBtc);
+        tradeFeeInBsqToggle.setSelected(!currencyForMakerFeeBtc);
+
+        if (!DevEnv.isDaoActivated()) {
+            tradeFeeInBtcToggle.setVisible(false);
+            tradeFeeInBtcToggle.setManaged(false);
+            tradeFeeInBsqToggle.setVisible(false);
+            tradeFeeInBsqToggle.setManaged(false);
+        }
```

Comment:
I guess all those trade fee UI components won't be necessary anymore for atomic BSQ trades.